PowerShell/sonar-lang-defs.txt

```diff
@@ -15,7 +15,6 @@ C
     filter rm_comments_in_strings " /* */
     filter rm_comments_in_strings " // 
     filter call_regexp_common C++
-    extension c
     extension C
```

Comment:
Should you keep `extension c` and not `extension C`?

Reply:
Done.